src/snapred/backend/dao/ingredients/VanadiumReductionIngredients.py

```diff
@@ -1,7 +1,7 @@
 from pydantic import BaseModel
 
+from snapred.backend.dao.ingredients.SmoothDataExcludingPeaksIngredients import SmoothDataExcludingPeaksIngredients
```

Comment:
you can simplify these further even
```
from snapred.backend.dao.ingredients import SmoothDataExcludingPeaksIngredients
```

Reply:
In this exact place you mentioned it, this creates type-checking errors.  However, outside of pydantic definitions this works well, and simplifies some of the import statements with multiple ingredients.